### DISCUSSION TITLE
Content inside Custom MenuList not focusable

### DISCUSSION THREAD

**Discussion by Cuttsy27** [JedWatson/react-select#4532] [Q&A]
Hi, 

I'm trying to add functionality to a reusable SelectMenu component in my project and one of the features should be to add custom content above the options list (inside MenuList?). I can render content above the menu's options here, but it isn't focusable/interactable. Adding the content to the Menu component closes the SelectMenu on click/focus also.

I have seen the experimental popout example in the docs, but I assume that only allows the default search input and probably removes other functionality from properly displaying such as multi-select.

**Comment by ebonow** [comment]
Greetings @Cuttsy27 ,

There are a few things involved here.

1) The default behavior of the select is to close the menu when the input is blurred. 
2) react-select uses clicks outside of the control or the menuList as indicators that the menu should also be closed (I have shelved fix for this to look at clicks inside of the Menu ref instead of the MenuList ref)

Before looking at possible solutions, what kind of content are you looking at adding? Is it similar to one of your options or some other content?

**Reply by cubecleveland** [reply]
Having the same issue..... Was that ever solved ? Trying to put an input box inside the menu itself.....
